examples/deviceshifu/demo_device/edgedevice-opcua/requirements.txt

```diff
@@ -0,0 +1,2 @@
+asyncua
```

Comment:
let's use `examples/opcuaDeviceShifu/server/requirements.txt`, can remove `asyncua` now since we are not using it anymore